contracts/modules/ReserveAuction/V1/ReserveAuctionV1.sol

```diff
@@ -14,8 +14,7 @@ import {ModuleNamingSupportV1} from "../../../common/ModuleNamingSupport/ModuleN
 /// @author tbtstl <t@zora.co>
 /// @notice This contract allows users to list and bid on ERC-721 tokens with timed reserve auctions
 contract ReserveAuctionV1 is ReentrancyGuard, UniversalExchangeEventV1, IncomingTransferSupportV1, FeePayoutSupportV1, ModuleNamingSupportV1 {
-    /// @dev The indicator to pass all remaining gas when paying out royalties
-    uint256 private constant USE_ALL_GAS_FLAG = 0;
+    uint256 private constant OUTGOING_TRANSFER_GAS_LIMIT = 30000;
```

Comment:
@kulkarohan - When this was originally written in https://github.com/ourzora/v3/pull/122/commits/5511daf935f623b6f402926d2040c00f461ee9c6, was `30000` chosen for a particular reason over another number? I also wonder about the [gas limit for handling a royalty payout](https://github.com/ourzora/v3/pull/122/commits/5511daf935f623b6f402926d2040c00f461ee9c6#diff-55d23cbd4dbeccddcd78cfdfd565cd9bca4e1539719eb60346e56959d39b89d2R491) being set at `200000`. 

Here to learn! 💭

Reply:
hey @almndbtr, the numbers are rough estimates based on the gas usages tested for `_handleOutgoingTransfer` and `_handleRoyaltyPayout`. we're bumping both up to be safe, as its difficult to predict every edge-case ahead of time. a PR will be incoming soon taking care of this and removing the variable, so feel free to close this. but thanks for bringing it up!

Reply:
Thank you! Will close now ✌🏽